src/btree/bt_debug.c

```diff
@@ -645,7 +626,7 @@ __debug_cell_kv(
     if (F_ISSET(ds, WT_DEBUG_UNREDACT))
         WT_RET(ds->f(ds, "\t%s: len %" PRIu32, __wt_cell_type_string(unpack->raw), unpack->size));
     else
-        WT_RET(ds->f(ds, "\t%s: {REDACTED}", __wt_cell_type_string(unpack->raw)));
+        WT_RET(ds->f(ds, "\t%s: REDACTED", __wt_cell_type_string(unpack->raw)));
```

Comment:
I'd like to keep the curly braces here so it's identical to unredacted behaviour. Other than that looking good!

Reply:
done d1f62736d98dead640b595af67e2dea7aad7a8a2 ty